images/server/Containerfile.fedora

```diff
@@ -17,6 +17,10 @@ ARG INSTALL_CUSTOM_REPO=
 
 MAINTAINER John Mulligan <jmulligan@redhat.com>
 
+LABEL org.opencontainers.image.title="Samba container"
+LABEL org.opencontainers.image.description="Samba container"
+LABEL org.opencontainers.image.vendor="Samba in Kubernetes"
```

Comment:
Nice! Those annotation are good idea, and should probably be added to samba-operator's Container as well (naturally, not relevant to this review).